_test_/index.test.tsx

```diff
@@ -1,9 +1,18 @@
-﻿import React from 'react';
+﻿// @ts-nocheck
```

Comment:
We don't really want this - this will turn off typescript for the whole file.
We'd ideally just turn it off for a single line, so that we still get all the goodness elsewhere in the file.

```ts
// eslint-disable-next-line
// @ts-ignore
mockNextUseRouter.mockReturnValue({query: {token: "100"}});
```

Reply:
I'll try that again. last night circle ci was still failing when I added those two lines

Reply:
I think you added them the wrong way around?